README.md

```diff
@@ -89,6 +89,10 @@ Of course there are more [examples for you to peruse](examples/). You could also
 2. `script/release`
 3. Profit.
 
+### Development in docker
```

Comment:
Sure, I am not e big expert in docker just use it in our testing/development suits. But I think it will be fun to learn new resoanable things in future PRs.